playground/__tests__/Components.test.tsx

```diff
@@ -24,16 +25,16 @@ describe("ActionButton component", () => {
 /** Testing the Dropdown Component */
 describe("Dropdown Component", () => {
   it("should change the selected example when clicked", async () => {
-    const { getByRole, getAllByRole, getByTestId } = render(<DropDown />);
+    const { getByRole, getByTestId } = render(<DropDown />);
 
-    fireEvent.mouseDown(getByRole("button"));
-    const listbox = within(getByRole("listbox"));
+    fireEvent.mouseDown(getByRole('button'));
+    const listbox = within(getByRole('listbox'));
 
-    fireEvent.click(listbox.getByText("comparison"));
-    expect(getByTestId("listbox")).toHaveTextContent("comparison");
+    fireEvent.click(listbox.getByText("example"));
+    expect(getByTestId('listbox')).toHaveTextContent('example');
```

Comment:
the quotes are inconsistent ... please check that you're running your formatter properly

here is how to setup prettier: https://prettier.io/docs/en/precommit.html

Reply:
done, will set up